pre_commit/main.py

```diff
@@ -122,6 +123,11 @@ def main(argv=None):
     if len(argv) == 0:
         argv = ['run']
     args = parser.parse_args(argv)
+    if hasattr(args, 'files') and args.files:
```

Comment:
I guess I was wrong here, it'll only have that attribute if `args.command == 'run'`, I'd prefer that over the hasasttr check :)